admin/rtd-checkout-main

```diff
@@ -0,0 +1,6 @@
+# See .readthedocs.yml
+set -ex
+if git symbolic-ref HEAD; then
+  git checkout origin/main -- doc/releases
+  git status
```

Comment:
Dumping "git status" would be useful in all cases IMO, so I'd move after `fi`.  Otherwise LGTM.